godtools_api/templates/default/godtools_api_properties.xml.erb

```diff
@@ -17,6 +17,7 @@
     <entry key="memcachedEnabled"><%= node['godtools']['memcached_enabled'] %></entry>
     <entry key="memcachedHost"><%= node['opsworks']['layers']['memcached']['instances']['memcached1']['private_ip'] %></entry>
     <entry key="memcachedPort"><%= node['godtools']['memcached_port'] %></entry>
+    <entry key="GoogleApiKey"><%= nod['godtools']['google_api_key'] %></entry>
```

Comment:
minor nitpick, but can the 'G' be lowercase so it matches the format of the rest of the properties?